src/App.tsx

```diff
@@ -404,6 +405,26 @@ const MainStack = () => {
   );
 };
 
+const linking = {
+  prefixes: [
+    DEEP_LINK_PREFIX,
+    DEEP_LINK_DOMAIN
+  ],
+  config: {
+    [ScreenNames.CloseContactAlert]: '/alert',
```

Comment:
This link will show an alert screen even if the user has no alerts (with the date field missing). We've added checks to make sure we don't navigate to this screen without a current exposure. 

Maybe we could point this at an intermediate screen that returns `<Loading />` then in a useEffect navigates to `ScreenNames.CloseContactAlert` if there's an exposure, or to the default screen if there isn't?

![Screenshot_20201214-145249_COVID Alert NY](https://user-images.githubusercontent.com/29628323/102097085-5f5f8000-3e1d-11eb-806e-457cfa54aa4d.jpg)